GestionVeterinarias/Veterinarias/LogicaVeterinarias/Classes/Veterinaria.cs

```diff
@@ -4,9 +4,54 @@
 using System.Text;
 using System.Threading.Tasks;
 
+
 namespace LogicaVeterinarias.Classes
 {
     class Veterinaria
     {
+        public int Id { get; set; }
+        public string Nombre { get; set; }
+        public string Direccion { get; set; }
+        public string Telefono { get; set; }
+        public Dictionary<Veterinario> DiccionarioVeterinarios { get; }
+        public Dictionary<Cliente> DiccionarioClientes { get; }
+        public Dictionary<Consulta> DiccionarioConsultas { get; }
+
+        public Veterinaria() { }
+
+        public Veterinaria(string nombre, string direccion, string telefono)
+        { 
```

Comment:
lo mismo que quedamos como estaba:
public Veterinaria(int id, string nombre, string direccion, string telefono)
        { 
        this.id = id;